docs/examples/tests/guides/test_mock_ansatz.py

```diff
@@ -0,0 +1,21 @@
+import pytest
+from orquestra.vqa.api.ansatz_test import AnsatzTests
+
+from ...guides.mock_ansatz import MockAnsatz
+
+
+class TestMockAnsatz(AnsatzTests):
```

Comment:
Same as before, it's enough if example code snippets run.

Reply:
This file runs all the code snippets and also runs all the default tests for MockAnsatz.

Reply:
1. Which line executes code snippets, cause I totally don't see it?
2. No need to add tests for mocks 🤷‍♂️

Reply:
The import itself will run all the code snippets right?

Reply:
Seriously? Importing Python file executes it?

Reply:
Yeah. Everything that isn't in a function gets executed.

Reply:
Wow!
I think we should make it more explicit in future, but I guess it's fine for now 🤷‍♂️